pyproject.toml

```
@@ -0,0 +1,67 @@
+[project]
+name = "openvino"
+dynamic = ["version", "entry-points", "scripts"]
+description = "OpenVINO(TM) Runtime"
+readme = {file = "README.md", content-type="text/markdown"}
+license = {text = "OSI Approved :: Apache Software License"}
+authors = [
+    { name = "OpenVINO Developers", email = "openvino@intel.com" },
+]
+classifiers = [
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+requires-python = ">=3.9"
+
+dependencies = [
+    "numpy>=1.16.6,<2.2.0",
```

```suggestion
    "numpy>=1.16.6,<2.3.0",
```
original req: https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/requirements.txt